### DISCUSSION TITLE
Hide title on one page

### DISCUSSION THREAD

**Discussion by mayanayza** [GetPublii/Publii#1706] [Help] [answered]
I would like to hide the title on one page on my website (not a post).

Is there any way to do this? I've tried custom CSS but can't seem to find a selector that can be limited to a single page; I can only target all titles on all pages.

**Comment by candidexmedia** [comment]
My recommendation would be to edit your theme files, and make whether a title is displayed conditional using the [Theme Settings API](https://getpublii.com/dev/theme-settings-api/).

How comfortable are you with this?

**Comment by gpsblues** [comment, marked as answer]
Hi,

Since you know CSS you can edit the post template by inserting an id or class to the title that corresponds to the page id.
For exemple from this:
`<h1 class="original-class">{{title}}</h1>`
to this:
`<h1 id="my{{id}}" class="original-class my{{id}}">{{title}}</h1>`

Each page will have a unique id, class or both (eg. id="my23").

There are other ways such as:
- create a page template without a title that you use only for pages that you don't want to show the title.
- add a custom option to page to show or hide the title
- use the smart replace plugin to remove a specific html tag such as "`<h1>My title to hide</h1>`"

**Reply by candidexmedia** [reply]
The `{{id}}` solution is really clever!

**Reply by gpsblues** [reply]
:)

**Reply by gpsblues** [reply]
It is also interesting to use {{slug}} which creates unique ids like {{id}} but more user friendly. I use them, for example, as internal links to the home page or to a landing page that groups together multiple pages/posts.

**Reply by mayanayza** [reply]
I was able to edit the page ID in the post template and used custom CSS to target that, thank you!
